components/application-mgt/org.wso2.carbon.identity.application.mgt.ui/src/main/java/org/wso2/carbon/identity/application/mgt/ui/util/ApplicationMgtUIUtil.java

```diff
@@ -19,14 +19,65 @@
 package org.wso2.carbon.identity.application.mgt.ui.util;
 
 import org.wso2.carbon.identity.application.mgt.ui.ApplicationBean;
+
 import javax.servlet.http.HttpSession;
+import java.util.HashMap;
+import java.util.Map;
+import java.util.UUID;
 
 public class ApplicationMgtUIUtil {
-    static public ApplicationBean getApplicationBeanFromSession(HttpSession session, String spName) {
-        if (session.getAttribute(spName) == null) {
+
+    /**
+     * Get related application bean from the session.
+     *
+     * @param session HTTP Session.
+     * @param spName  Service provider name.
+     * @return ApplicationBean
+     */
+    public static ApplicationBean getApplicationBeanFromSession(HttpSession session, String spName) {
+
+        Map<String, UUID> spUniqueIdMap;
+
+        if (session.getAttribute("spUniqueIdMap") == null) {
```

Comment:
We can see "spUniqueIdMap" string used in few places in this class. Can we define a String constant for "spUniqueIdMap"